missing kernel image

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks! -->

When deploying the falco version .30.0 it receives an error when deploying on a instance or no kernel found at the location it is directed to pull from. The ec2 instance is a amazonlinux 2 instance. 

<!-- A clear and concise description of what the bug is. -->

* Running falco-driver-loader for: falco version=0.30.0, driver version=3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4
* Running falco-driver-loader with: driver=bpf, compile=yes, download=yes
* Mounting debugfs
* Trying to download a prebuilt eBPF probe from https://download.falco.org/driver/3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4/falco_amazonlinux2_5.4.209-116.367.amzn2.x86_64_1.o
curl: (22) The requested URL returned error: 404 
Unable to find a prebuilt falco eBPF probe
* Trying to compile the eBPF probe (falco_amazonlinux2_5.4.209-116.367.amzn2.x86_64_1.o)
make[1]: *** /lib/modules/5.4.209-116.367.amzn2.x86_64/build: No such file or directory.  Stop.
make: *** [Makefile:20: all] Error 2
mv: cannot stat '/usr/src/falco-3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4/bpf/probe.o': No such file or directory
Unable to load the falco eBPF probe
Wed Nov 16 18:51:44 2022: Falco version 0.30.0 (driver version 3aa7a83bf7b9e6229a3824e3fd1f4452d1e95cb4)
Wed Nov 16 18:51:44 2022: Falco initialized with configuration file /etc/falco/falco.yaml
Wed Nov 16 18:51:44 2022: Loading rules from file /etc/falco/falco_rules.yaml:
Wed Nov 16 18:51:45 2022: Loading rules from file /etc/falco/falco_rules.local.yaml:
Wed Nov 16 18:51:45 2022: Unable to load the driver.
Wed Nov 16 18:51:45 2022: Runtime error: can't open BPF probe '/root/.falco/falco-bpf.o': Errno 2. Exiting.``` 


**Environment**
AWS ECS on EC2 instance of amazon linux 2 

<!-- Please complete the following info. -->

- Falco version:
<!-- Use "falco --version". -->
- System info: .30
<!-- Falco has a built-in support command you can use  "falco --support | jq .system_info" -->
- Cloud provider or hardware configuration:AWS
- OS: amazon linux 2
<!-- Eg., output of "cat /etc/os-release". -->
- Kernel:
<!-- Eg., output of "uname -a". -->
- Installation method:
<!-- Eg., Kubernetes, RPM, DEB, from source? -->
ECS/helm
**Additional context**

<!-- Add any other context about the problem here. -->

--- a/falco/CHANGELOG.md
+++ b/falco/CHANGELOG.md
@@ -3,6 +3,10 @@
 This file documents all notable changes to Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v2.5.1
+
+* Update README to clarify driver behavior for chart
+
 ## v2.5.0
 
 * Support custom dictionaries when setting environment variables

--- a/falco/Chart.yaml
+++ b/falco/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: falco
-version: 2.5.0
+version: 2.5.1
 appVersion: 0.33.1
 description: Falco
 keywords:

--- a/falco/README.md
+++ b/falco/README.md
@@ -6,9 +6,9 @@
 
 The deployment of Falco in a Kubernetes cluster is managed through a **Helm chart**. This chart manages the lifecycle of Falco in a cluster by handling all the k8s objects needed by Falco to be seamlessly integrated in your environment. Based on the configuration in `values.yaml` file, the chart will render and install the required k8s objects. Keep in mind that Falco could be deployed in your cluster using a `daemonset` or a `deployment`. See next sections for more info.
 
-## Atention
+## Attention
 
-Before installing Falco in a Kubernetes cluster a user should check that the kernel version used in the nodes are supported by the community, also before reporting any issue with Falco (missing kernel image, CrashLoopBackOff and similar) make sure to read [about the driver](#about-the-driver) section and adjust your setup as required.
+Before installing Falco in a Kubernetes cluster, a user should check that the kernel version used in the nodes is supported by the community. Also, before reporting any issue with Falco (missing kernel image, CrashLoopBackOff and similar), make sure to read [about the driver](#about-the-driver) section and adjust your setup as required.
 
 ## Adding `falcosecurity` repository
 
@@ -53,9 +53,14 @@ Falco needs a **driver** (the [kernel module](https://falco.org/docs/event-sourc
 
 By default the drivers are managed using an *init container* which includes a script (`falco-driver-loader`) that either tries to build the driver on-the-fly or downloads a prebuilt driver as a fallback. Usually, no action is required.
 
-Kernel versions and flavors are automatically discovered by the kernel-crawler. At the time being, it runs weekly, on each Monday mornings. We have a site where users can check for the discovered kernel flavors and version, [example for Amazon Linux 2](https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=AmazonLinux2).
 
-The discovery of a kernel version by the [kernel-crawler](https://falcosecurity.github.io/kernel-crawler/) does not imply that pre-built kernel modules and bpf probes are available. That is because the jobs running on our test-infra runs periodically and are not in sync with the kernel-crawler runs. Please keep in mind that the building process is based on best effort. Users can check the existence of prebuilt modules at the following [link](https://download.falco.org/driver/site/index.html?lib=3.0.1%2Bdriver&target=all&arch=all&kind=all).
+##### Pre-built drivers
+
+The [kernel-crawler](https://github.com/falcosecurity/kernel-crawler) automatically discovers kernel versions and flavors. At the time being, it runs weekly. We have a site where users can check for the discovered kernel flavors and versions, [example for Amazon Linux 2](https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=AmazonLinux2).
+
+The discovery of a kernel version by the [kernel-crawler](https://falcosecurity.github.io/kernel-crawler/) does not imply that pre-built kernel modules and bpf probes are available. That is because once kernel-crawler has discovered new kernels versions, the drivers need to be built by jobs running on our [Driver Build Grid infra](https://github.com/falcosecurity/test-infra#dbg). Please keep in mind that the building process is based on best effort. Users can check the existence of prebuilt modules at the following [link](https://download.falco.org/driver/site/index.html?lib=3.0.1%2Bdriver&target=all&arch=all&kind=all).
+
+##### Building the driver on the fly (fallback)
 
 If a prebuilt driver is not available for your distribution/kernel, users can build the modules by them self or install the kernel headers on the nodes, and the init container (falco-driver-loader) will try and build the module on the fly.
 

--- a/falco/README.md
+++ b/falco/README.md
@@ -6,6 +6,10 @@
 
 The deployment of Falco in a Kubernetes cluster is managed through a **Helm chart**. This chart manages the lifecycle of Falco in a cluster by handling all the k8s objects needed by Falco to be seamlessly integrated in your environment. Based on the configuration in `values.yaml` file, the chart will render and install the required k8s objects. Keep in mind that Falco could be deployed in your cluster using a `daemonset` or a `deployment`. See next sections for more info.
 
+## Atention
+
+Before installing Falco in a Kubernetes cluster a user should check that the kernel version used in the nodes are supported by the community, also before reporting any issue with Falco (missing kernel image, CrashLoopBackOff and similar) make sure to read [about the driver](#about-the-driver) section and adjust your setup as required.
+
 ## Adding `falcosecurity` repository
 
 Before installing the chart, add the `falcosecurity` charts repository:
@@ -49,7 +53,13 @@ Falco needs a **driver** (the [kernel module](https://falco.org/docs/event-sourc
 
 By default the drivers are managed using an *init container* which includes a script (`falco-driver-loader`) that either tries to build the driver on-the-fly or downloads a prebuilt driver as a fallback. Usually, no action is required.
 
-If a prebuilt driver is not available for your distribution/kernel, Falco needs **kernel headers** installed on the host as a prerequisite to build the driver on the fly correctly. You can find instructions for installing the kernel headers for your system under the [Install section](https://falco.org/docs/getting-started/installation/) of the official documentation.
+Kernel versions and flavors are automatically discovered by the kernel-crawler. At the time being, it runs weekly, on each Monday mornings. We have a site where users can check for the discovered kernel flavors and version, [example for Amazon Linux 2](https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=AmazonLinux2).
+
+The discovery of a kernel version by the [kernel-crawler](https://falcosecurity.github.io/kernel-crawler/) does not imply that pre-built kernel modules and bpf probes are available. That is because the jobs running on our test-infra runs periodically and are not in sync with the kernel-crawler runs. Please keep in mind that the building process is based on best effort. Users can check the existence of prebuilt modules at the following [link](https://download.falco.org/driver/site/index.html?lib=3.0.1%2Bdriver&target=all&arch=all&kind=all).
+
+If a prebuilt driver is not available for your distribution/kernel, users can build the modules by them self or install the kernel headers on the nodes, and the init container (falco-driver-loader) will try and build the module on the fly.
+
+Falco needs **kernel headers** installed on the host as a prerequisite to build the driver on the fly correctly. You can find instructions for installing the kernel headers for your system under the [Install section](https://falco.org/docs/getting-started/installation/) of the official documentation.
 
 #### About Plugins
 [Plugins](https://falco.org/docs/plugins/) are used to extend Falco to support new **data sources**. The current **plugin framework** supports *plugins* with the following *capabilities*:

